verticapy/performance/vertica/collection/collection_tables.py

```diff
@@ -139,8 +153,37 @@ def getAllCollectionTables(
     key: str
         A suffix for all table names to help uniquely identify the table.
     """
+    if version == BundleVersion.V1:
+        return _getAllTables_v1(target_schema, key)
+
+    raise ValueError(f"Unrecognized bundle version {version}")
+
+
+ALL_TABLES_V1 = [
+    AllTableTypes.COLLECTION_EVENTS,
+    AllTableTypes.COLLECTION_EVENTS,
```

Comment:
Have you repeated this element on purpose? Is it the reason you have created the ALL_TABLES_V1 list instead of iterating over AllTableTypes?
A line of clarifying in-code comment would be nice.

Reply:
I'll add a comment in the code to clarify.

The reason is: we know that we will change the version of the bundle, and that future versions will have more tables. So we need to version the list of tables that we expect in a bundle. It's overkill now: there is only one version. But I expect the bundle versions to change soon when we collect more files.

Reply:
It makes sense for creating the `ALL_TABLES_V1` list. What about repeating `AllTableTypes.COLLECTION_EVENTS` in the list?

Reply:
Good catch! There is an unwanted dupe. Fixed in d87dc12b, test added in 4d861c4b